Marlin/pins_RAMPS_14.h

```diff
@@ -247,7 +248,8 @@
     #endif
   #else // !NEWPANEL (Old-style panel with shift register)
 
-    #define BEEPER_PIN 33   // No Beeper added
+    // No Beeper added
+    #define BEEPER_PIN 33
```

Comment:
A did the fastest quick fix I had on hand..
If we find a better one I will revert all this garbage back.